.travis.yml

```diff
@@ -36,6 +36,7 @@ install:
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
   - travis_install_nightly
   - pip install beautifulsoup4
```

Comment:
I don't found where you are using `bs4`
Could we take advantage of this PR to remove it?